src/w2grid.js

```diff
@@ -674,6 +674,7 @@
             // process sort
             this.records.sort(function (a, b) {
                 var ret = 0;
+                var defaultToString = {}.toString;
```

Comment:
This syntax "{}.toString" is very confusing to me. I am not sure I understand what you are trying to do with it. Can you please explain /or rewrite in more conservative way.